static/app/views/externalRedirect/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import dompurify from 'dompurify';
+
+import Panel from 'sentry/components/panels/panel';
+import {t, tct} from 'sentry/locale';
+import {isUrl} from 'sentry/utils';
+
+function ExternalRedirect() {
+  const [count, setCount] = useState<number>(5);
+  const [redirectUrl, setRedirectUrl] = useState<string | null>(null);
+  const [isError, setIsError] = useState(false);
```

Comment:
I don't think you need to keep this in state. If you extract the close tab logic and the `isError` block to a new component, you could simply do:

```jsx
const isValidUrl = isUrl(url)

if (!isUrl(url)) {
	return <InvalidUrlCloseTab />
}
```